doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -190,72 +215,73 @@ NFC samples
 
     * Skip NDEF content printing when message parsing returns an error.
 
-nrf5340 Samples
----------------
 
-* Added:
+OpenThread samples
+------------------
 
-  * :ref:`nrf5340_remote_shell` sample.
-  * :ref:`nrf5340_multicore` sample.
+* Updated:
 
-nrf9160 Samples
----------------
+  * The relevant sample documentation pages with information about support for :ref:`Trusted Firmware-M <ug_tfm>`.
+  * :ref:`ot_cli_sample` sample:
+
+    * Added:
+
+      * :file:`prj_thread_1_2.conf` to support Thread v1.2 build for the nRF52 and nRF53 Series devices.
+      * Child image configuration files for network core builds for Thread v1.2 build.
+
+  * All sample documentation with a Configuration section, and organized relevant information under that section.
```

Comment:
I think we're making this too complicated with these "Added", "Updated" and so on embedded under other "Added", "Updated". "Added" was originally meant as adding a new functionality. If something that we already have is being extended with new features, I don't think it makes sense to list this addition as "Added" -- it's just a plain update to the implemented feature.

Put simple: It is just harder to read the changelog with so many "Added" and "Updated" embedded in one another.

Also, single entries do not need list formatting. We removed single bullet point entries from elsewhere in the docs, so I don't see why we shouldn't be consistent about this in the changelog.

Reply:
Discussed with  @greg-fer. Retaining the changes for now, will follow up with more changes if needed after a discussion later.